OrbitGl/TimeGraph.cpp

```diff
@@ -196,6 +196,34 @@ void TimeGraph::ZoomTime(float a_ZoomValue, double a_MouseRatio) {
   SetMinMax(minTimeUs, maxTimeUs);
 }
 
+void TimeGraph::VerticalZoom(float zoom_value, double mouse_relative_position) {
+  static double increment_ratio = 0.1;
+
+  double scale = zoom_value > 0 ? 1 + increment_ratio : 1 - increment_ratio;
+
+  float world_height = m_Canvas->GetWorldHeight();
+  double y_mouse_position = m_Canvas->GetWorldTopLeftY() - mouse_relative_position * world_height;
+  double top_distance = m_Canvas->GetWorldTopLeftY() - y_mouse_position;
+
+  // As we are scaling things, every item is going to scale its position.
```

Comment:
This comment is very general and doesn't really help understanding the line below. Consider removing it altogether.